charts/mattermost-enterprise-edition/values.yaml

```diff
@@ -127,8 +127,36 @@ mattermostApp:
       # - secretName: chart-example-tls
       #   hosts:
       #     - mattermost.example.com
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/proxy-buffering: "on"
+      nginx.ingress.kubernetes.io/proxy-body-size: 50m
+      nginx.ingress.kubernetes.io/proxy-send-timeout: "600"
+      nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
+      nginx.ingress.kubernetes.io/configuration-snippet: |
+        proxy_cache mattermost_cache;
+        proxy_cache_revalidate on;
+        proxy_cache_min_uses 2;
+        proxy_cache_use_stale timeout;
+        proxy_cache_lock on;
 
-  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 2
+    targetMemoryUtilizationPercentage: 50
+    targetCPUUtilizationPercentage: 50
+
+  antiAffinity:
+    enabled: false
+    # can be hard/soft, affects weight
+    mode: ""
+    # can implement self affinty, add affinity at the correct location
+    custom:
```

Comment:
this for me is more confusing.

i still prefer the user to define their own affinity and not have anything predefined.
but for now lets go with the predefined one you made @erezo9, can you please remove the custom part? thanks for working on this

Reply:
ye sure ill commit the change and ill message you about